src/main/java/com/nike/cerberus/command/core/CreateDatabaseCommand.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.core;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.beust.jcommander.ParametersDelegate;
+import com.nike.cerberus.command.Command;
+import com.nike.cerberus.domain.cloudformation.TagParametersDelegate;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.operation.core.CreateDatabaseOperation;
+
+import static com.nike.cerberus.command.core.CreateDatabaseCommand.COMMAND_NAME;
+
+/**
+ * Command for creating the base components for Cerberus.
+ */
+@Parameters(commandNames = COMMAND_NAME,
+        commandDescription = "Create the database to be used by the Cerberus Management Service (CMS)")
+public class CreateDatabaseCommand implements Command {
+
+    public static final String COMMAND_NAME = "create-database";
+
+    @Parameter(names = "--cms-db-port",
```

Comment:
Seems like DB port should have a default and then it should be exported as a parameter so that it can be auto picked up by CMS stack.

Reply:
Or maybe it already defaults and that is why it is capital Integer